doc/man/scons.xml

```diff
@@ -3952,50 +3952,51 @@ env = conf.Finish()
 <title>Command-Line Construction Variables</title>
 
 <para>Often when building software,
-some variables must be specified at build time.
+some variables need to be specified at build time.
 For example, libraries needed for the build may be in non-standard
 locations, or site-specific compiler options may need to be passed to the
 compiler.
-&scons;
-provides a <firstterm>&Variables;</firstterm>
+&SCons;
+provides a <firstterm>Variables</firstterm>
```

Comment:
?why remove &

Reply:
as an attribute it's a reference to the function. in this context that seems not the intent (and its clearly pretty minor)

Reply:
changed back.